modules/@apostrophecms/import-export-page/index.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  improve: '@apostrophecms/page',
+
+  utilityOperations (self) {
+    if (self.options.import === false) {
```

Comment:
From the ticket, the option name is `shareDocsDisableImport`. Not sure it's a good one since he module is called `import-export` but we might check.